iron-location.js

```diff
@@ -379,6 +379,15 @@ Polymer({
     if (normalizedHref[0] !== '/') {
       normalizedHref = '/' + normalizedHref;
     }
+    
+    /**
+     * Fix for iOS 10. It adds extra `/` to pathname when create a new URL()
+     */
+    var path = normalizedHref;
+    var pathHasSlashes = path.indexOf("///");
```

Comment:
You can probably use a regexp for that. Might be faster
```suggestion
    normalizedHref = path.replace(/^\/\/\//, '/');
```

Reply:
@paulvarache It makes sense for me. I replaced string method by regExp